write-fonts/src/tables/variations/ivs_builder.rs

```diff
@@ -17,11 +17,14 @@ type TemporaryDeltaSetId = u32;
 ///
 /// This handles assigning VariationIndex values to unique sets of deltas and
 /// grouping delta sets into [ItemVariationData] subtables.
-#[derive(Clone, Default, Debug)]
+#[derive(Clone, Debug)]
 pub struct VariationStoreBuilder {
     // region -> index map
     all_regions: HashMap<VariationRegion, usize>,
     delta_sets: DeltaSetStorage,
+    // must match fvar. We require the user to pass this in because we cannot
+    // infer it in the case where no deltas are added to the builder.
+    axis_count: u16,
```

Comment:
If we don't already we should probably validate that the regions match this count since that's possible to check with access to only self?

Reply:
yea let's open an issue for that, I'm going to be done for the day shortly.